docs/provision_hypershift_clusters_by_manifestwork.md

```diff
@@ -0,0 +1,384 @@
+# Creating a hosted cluster on a remote hosting cluster without HypershiftDeployment CR
+
+## How HypershiftDeployment creates a hosted cluster on a remote hosting cluster
+
+When you create a HypershiftDeployment CR, the HypershiftDeployment operator creates a manifestwork CR in the target managed (hypershift hosting) cluster's namespace on the ACM hub cluster. The payload of the manifestwork CR includes:
+
+- the namespace where the hosted cluster is created
+- HostedCluster CR
+- NodePool CRs
+- Pull secret
+- Control plane operator credential secret
```

Comment:
I am pretty sure we don't need the three secrets, since you now put the values of those secrets as ARN's in the roles section on the HostedCluster